content/articles/gulp-sass-2.md

```diff
@@ -0,0 +1,124 @@
+Title: gulp-sass v2 released
+Date: 2015-05-06
+Short_summary: showing what's new in gulp-sass but most importantly in libsass
+Category: Dev
+Authors: Vincent
+
+*Reading time: ~15 minutes.*
+
+
+[gulp-sass](https://github.com/dlmanning/gulp-sass) v2 was released today!  
+In itself, this is not a big change as this is simply a wrapper for node-sass which uses libsass so, while this version is a complete rewrite this is not the important news. The important news is that it ships with [node-sass](https://github.com/sass/node-sass) v3 which is using [libsass](http://libsass.org/) 3.2.2.
+<!-- PELICAN_END_SUMMARY -->
+
+We will see in a short moment why it is exciting but first a small explanation on what is sass and libsass.  
+
+**TL;DR**:
+```bash
+$ rm -rf node_modules/gulp-sass/
+$ npm install gulp-sass --save-dev
+// enjoy nearly full sass compatibility
+```
+
+## Sass
+Nowadays, most front-end developers and designers use CSS preprocessors to (immensely) simplify their work.  
+A CSS preprocessor brings variables, nesting, mixins, functions and other things to CSS.  Let's see a quick example of why we would use one, in this case using variables for colours:
+
+```css
+.header {
+  background: #484c55;
+}
+
+.header .title {
+  background: #5a5d66; /* 10% lighter than #484c55 */
+}
+
+.header .title .logo {
+  background: #484c55;
+}
+```
+
+And in Sass:
+```sass
+// functions.scss
+/// Slightly lighten a color
+@function tint($color, $percentage) {
+  @return mix(white, $color, $percentage);
+}
+
+// colors.scss
+@import "functions";
+
+$header-background: #484c55;
+$header-title-background: tint($header-background, 10%);
+
+// header.scss
+@import "colors";
+
+.header {
+  background: $header-background;
+
+  .title {
+    background: $header-title-background;
+
+    .logo {
+      background: $header-background;
+    }
+  }
+}
+```
+
+> The above ignores best practices about naming/nesting/specificity of selectors for the sake of example.
+
+While the Sass version is much longer, it makes your CSS much more readable, easy to organise and consistent. This is only a tiny part of what CSS preprocessors but it hopefully gives you an idea on why they are extremely useful.  
+
+[Sass](http://sass-lang.com/) is one of them. There is also [less](http://lesscss.org/), [stylus](http://learnboost.github.io/stylus/) and a few others but I personally use Sass so I will be talking about that.  
+
+Sass is written in Ruby, which means that Ruby is now needed to build your frontend. Ruby is also not the fastest programming language around and the compilation was taking over 10 seconds for some big projects, which is obviously not really sustainable when working.  
```

Comment:
"compilation was taking more than 10 seconds for some our larger projects"

I'd also skip words like "much more" (more is enough) or "obviously" (because it's not obvious to me :)